db/migrate/20210923215047_add_archived_to_lsg_bodies.rb

```diff
@@ -0,0 +1,5 @@
+class AddArchivedToLsgBodies < ActiveRecord::Migration[6.1]
+  def change
+    add_column :lsg_bodies, :archived, :boolean, default: false
```

Comment:
Should we also add a filed `archived_at` so that we could figure out when the record was archived?

Reply:
Should we keep record of all archive actions when was archived and when was unarchived like we have logs?

Reply:
We'll then have :archived, :archived_at fields. Won't it be ideal to keep only one of these? Both the fields can serve as a flag. So should we only go with the :archived_at field?